### PR TITLE
Nuclia Workflow Design

### PR DESCRIPTION
**Pull request by justdaksh**
1. Implemented `upload.py` 
2. Drafted `workflow` for the same
Additional context- 
https://github.com/collective/plone-nuclia/issues/3
https://github.com/collective/plone-nuclia/issues/2
cc: @stevepiercy @ebrehault @fredvd

### CODE REVIEW

File: submodules/volto
Comment (stevepiercy) [inline submodules/volto:1]: This file should not be committed to the repo. I do that upon a release of docs.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Looks good, except for the submodule commit. That needs to be removed from this PR. Thank you!

File: .github/workflows/nuclia_sync.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
+    - name: Install requirements
+      run: |
+        pip install -q -r requirements-initial.txt
+        pip install -q -r requirements.txt
+    - name: Run Nuclia Sync
```
Comment (stevepiercy) [inline .github/workflows/nuclia_sync.yml]: Let's use consistent spacing between `name`s. I think a blank line between each pair is helpful to see each task.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
A tiny style preference, then this is good to merge.

**Review by stevepiercy — APPROVED** [review]
This looks good.

**Comment by justdaksh** [comment]
@stevepiercy Going ahead with the merge.

**Comment by stevepiercy** [comment]
Woo-hoo! Thanks for verifying that you have sufficient permission. Next let's poke @fredvd to see what he needs from you to establish a staging server.

**Pull request merged**